advice_animal/cli.py

```diff
@@ -269,9 +269,14 @@ def show_list(ctx: click.Context) -> bool:
     for advice_name, check_cls in runner.order_check_classes(filter=everything):
         if check_cls.confidence.name != "UNSET":
             name = click.style(advice_name, fg=check_cls.confidence.name.lower())
+            description = click.style(check_cls.__doc__, fg=check_cls.confidence.name.lower()) if check_cls.__doc__ else None
         else:
             name = click.style(advice_name, fg="green")
+            description = click.style(check_cls.__doc__, fg="green") if check_cls.__doc__ else None
         click.echo(f"* {name}{' - (preview)' if check_cls.preview else ''}")
+        if description:
+            first_line = description.strip().split('\n')[0]  # Needs to be on a separate line to prevent backslash in f-string
+            click.echo(f"    {first_line}")
```

Comment:
Why only first line?

I think we should do this first line extraction before it is put through the `click.style()` call. Because it gets surrounded by escape codes for terminal colors.

Reply:
I was afraid it would look ugly to put entire docstrings in the bulleted advice list, but I'm not glued to the idea. What do you think of putting the descriptions behind a flag instead? So leave the bulleted list as-is, but something like `advice-animal click_version_option --describe` would return the docstring for that `Check` class. And thanks for catching the `click.style()` issue!

Reply:
No. Let's not add a new arg. I'm ok with the current implementation if you move the first line extraction to be before the click.style() call.